### DISCUSSION TITLE
Canʻt configure Plausible integration for my docs site

### DISCUSSION THREAD

**Discussion by liliakai** [GitbookIO/community#793] [0. General bugs and questions]
Frontend crashes consistently while accessing https://app.gitbook.com/o/WQpOq5ZFue4N6m65QCJq/integrations/plausible/configuration/site/site_ZWiTf

<img width="842" alt="Screenshot 2024-11-08 at 19 09 47" src="https://github.com/user-attachments/assets/38af5532-d606-4b79-9bcb-54e19450f160">

**Comment by emmerich** [comment]
That one should've been fixed too @liliakai if you could double check 🙏

**Comment by liliakai** [comment]
Still crashing for me.

**Comment by liliakai** [comment]
Update: now when I try to access it I get redirected to https://app.gitbook.com/o/<orgID>/integrations/plausible/about

**Reply by valentin0h** [reply]
@liliakai, we're migrating to a new UX for managing integrations; hence, there have been a few glitches over the past few days. This one should be fixed now.

**Reply by liliakai** [reply]
Well I was able to load the configuration screen but it crashed again as soon as I set the domain url.

**Comment by liliakai** [comment]
Maybe a separate issue but when Plausible.io tries to verify the site installation it says:
> We encountered an issue with your site's CSP. Please add plausible.io domain specifically to the allowed list of domains in your Content Security Policy (CSP)

**Comment by liliakai** [comment]
Please reopen. Itʻs still crashing consistently for me.

**Comment by liliakai** [comment]
Classic type exception guys letʻs gooooo
<img width="986" alt="Screenshot 2024-11-13 at 11 44 45" src="https://github.com/user-attachments/assets/f3f85117-6705-4fd7-b414-6c89344a24ab">

**Reply by valentin0h** [reply]
Have you tried hard reloading the app? I can see the configuration fine on my end.

**Comment by liliakai** [comment]
I even turned off my ad blockers so you can find it in sentry now:
> reporting exception to Sentry with event ID 1d019e7b0f8a47bb856bf0df73ea6e4b
